samples/client/petstore/dart-jaguar/flutter_petstore/openapi/lib/model/pet.dart

```diff
@@ -1,9 +1,9 @@
 import 'package:jaguar_serializer/jaguar_serializer.dart';
 
 
-import 'package:openapi/model/tag.dart';
+import 'package:openapi/model/package:openapi/model/tag/dart.dart';
```

Comment:
@nickmeinhold as discussed, we'll probably need to update the dart jaguar template to fix this.

Please take your time and let me know if you need help.

Reply:
Thanks @wing328, I've put up a fix but having trouble with the CI - I have already run `./bin/dart-jaguar-petstore.sh` and `./bin/utils/ensure-up-to-date --batch` locally, but neither produces any changes. It seems weird but this branch has been doing some really weird stuff.  I could start a new branch again, I don't see why that would help but maybe worth a try?

Reply:
Before running `./bin/dart-jaguar-petstore.sh`, did you run `mvn clean package` to recompile the JAR? I think you'd but asking just to make sure.

Reply:
Btw `./bin/utils/ensure-up-to-date --batch` (batch mode) is not ready yet and we've switched back to just `./bin/utils/ensure-up-to-date`

Reply:
Ugh, no that was totally it. I guess multi-tasking does make you stupid ;-) Thanks @wing328, good catch.

I think maybe the failure on Travis is unrelated...? 

Thanks for the heads up re: batch mode.  I was using batch mode because it prompts you to when you run just `./bin/utils/ensure-up-to-date` - maybe that's been changed already?  I'll grab the latest master and check.

Reply:
Just checked, running `./bin/utils/ensure-up-to-date` still says the following: 
```
IMPORTANT: this script should be run by the CI (e.g. Shippable) to ensure that the 'samples/' folder is up to date.
When running locally execute with argument --batch.
```
Want me to create an issue to change it?